src/infra/core/tests/tests/auth/test_authentication_service.rs

```diff
@@ -33,7 +33,7 @@ async fn test_enabled_login_methods() {
     let authentication_service = catalog.get_one::<dyn AuthenticationService>().unwrap();
 
     let mut supported_login_methods = authentication_service.supported_login_methods();
-    supported_login_methods.sort();
+    supported_login_methods.sort_unstable();
```

Comment:
Not sure I understand the rule.

Reply:
> **What it does**
> 
> When sorting primitive values (integers, bools, chars, as well as arrays, slices, and tuples of such items), it is typically better to use an unstable sort than a stable sort.
> 
> **Why is this bad?**
> 
> Typically, using a stable sort consumes more memory and cpu cycles. Because values which compare equal are identical, preserving their relative order (the guarantee that a stable sort provides) means nothing, while the extra costs still apply.
(c) https://rust-lang.github.io/rust-clippy/master/index.html#stable_sort_primitive

I had doubts about this rule at first, but since it is used for primitive values, I decided to leave it enabled